docker/etna-base-dev/entrypoints/build.sh

```diff
@@ -34,6 +34,10 @@ mkdir -p /usr/opt/httpd.conf.d
 echo "@app_name = '${APP_NAME}'" > /usr/opt/vars.rb
 rm -rf /usr/opt/httpd.conf.d/*.include
 
+# Prepare the puma.sh into the /app/bin directory.
+ln -sf /entrypoints/puma.sh /app/bin/
+chmod +x /app/bin/puma.sh
```

Comment:
Rather than every app individually maintaining puma.sh files, I've decided to settle on a single implementation from the `docker/etna-base-dev/entrypoints/puma.sh` file, and link it to the right place inside the container at build time.